antora-ui-camel/src/css/doc.css

```
@@ -36,6 +36,10 @@
   margin: 1.5rem 0;
 }
 
+.doc > h1.page:first-child img {
+  width: 1rem;
```

I've previewed the website locally. This looks a bit odd.. I think `1em` looks nicer.

```suggestion
  width: 1em;
```

---

Seems like `1em` is computed in `36px`, so that would be `2rem` here. I can push this change to this branch.

---

Done in 54034264945447cba2fdec1c4487be926ee77678